lib/Wikia/src/Service/User/Preferences/UserPreferences.php

```diff
@@ -68,16 +68,29 @@ public function get($userId, $pref, $default = null, $ignoreHidden = false) {
 	 * @param string $pref
 	 * @param string $val
 	 */
-	public function set($userId, $pref, $val) {
-		$this->load($userId);
+	public function set( $userId, $pref, $val ) {
+		$this->setMultiply( $userId, [ $pref => $val ] );
+	}
 
-		$default = $this->getFromDefault($pref);
-		if ($val == null && isset($default)) {
-			$val = $default;
+	/**
+	 * @param int $userId
+	 * @param array $prefs
+	 */
+
+	public function setMultiply( $userId, $prefs ) {
```

Comment:
Do you mean `setMultiple` here? Also whitespace between phpdoc and function signature